phpBB/docs/events.md

```diff
@@ -2227,6 +2227,14 @@ viewforum_body_topic_row_prepend
 * Since: 3.1.7-RC1
 * Purpose: Add content at the end of the topic list item.
 
+viewforum_body_topicrow_before
```

Comment:
To be consistent with the other events regarding the topicrow, we should probably change it to `viewforum_body_topic`_row_before` and move it accordingly before the entries above this one to have a proper alphabetical order.